docs/readmes/Makefile

```diff
@@ -7,8 +7,8 @@ FILES_TO_IGNORE = --ignore proposals/p010_vendor_neutral_dp.md \
 
 precommit:
 	docker build -t magma_readmes .
-	docker-compose run readmes markdownlint $(FILES_TO_IGNORE) . && echo PASSED
+	docker compose --compatibility run readmes markdownlint $(FILES_TO_IGNORE) . && echo PASSED
 
 precommit_fix:
```

Comment:
Not working for me:

```
lars@lars-Precision-5560:~/magma/docs (pr/redo_docker_compose_v2)$ make precommit_fix
make -C readmes precommit_fix
make[1]: Entering directory '/home/lars/magma/docs/readmes'
docker build -t magma_readmes .
Sending build context to Docker daemon  50.14MB
Step 1/4 : FROM node:14
 ---> c8e8caffee0b
Step 2/4 : RUN mkdir -p /readmes
 ---> Using cache
 ---> 9eb099b3efe4
Step 3/4 : WORKDIR /readmes
 ---> Using cache
 ---> b6274e456efb
Step 4/4 : RUN npm install --global markdownlint-cli
 ---> Using cache
 ---> 621d17a73bf1
Successfully built 621d17a73bf1
Successfully tagged magma_readmes:latest
docker compose --compatibility run readmes markdownlint --fix --ignore proposals/p010_vendor_neutral_dp.md --ignore proposals/p010_subscriber_scaling.md --ignore proposals/p015_tech_debt_week.md --ignore proposals/p018_control_network_metrics.md . && echo PASSED
unknown flag: --compatibility
See 'docker --help'.

Usage:  docker [OPTIONS] COMMAND

A self-sufficient runtime for containers

Options:
      --config string      Location of client config files (default "/home/lars/.docker")
  -c, --context string     Name of the context to use to connect to the daemon (overrides DOCKER_HOST env var and default context set with "docker context use")
  -D, --debug              Enable debug mode
  -H, --host list          Daemon socket(s) to connect to
  -l, --log-level string   Set the logging level ("debug"|"info"|"warn"|"error"|"fatal") (default "info")
      --tls                Use TLS; implied by --tlsverify
      --tlscacert string   Trust certs signed only by this CA (default "/home/lars/.docker/ca.pem")
      --tlscert string     Path to TLS certificate file (default "/home/lars/.docker/cert.pem")
      --tlskey string      Path to TLS key file (default "/home/lars/.docker/key.pem")
      --tlsverify          Use TLS and verify the remote
  -v, --version            Print version information and quit

Management Commands:
  app*        Docker App (Docker Inc., v0.9.1-beta3)
  builder     Manage builds
  buildx*     Docker Buildx (Docker Inc., v0.9.1-docker)
  config      Manage Docker configs
  container   Manage containers
  context     Manage contexts
  image       Manage images
  manifest    Manage Docker image manifests and manifest lists
  network     Manage networks
  node        Manage Swarm nodes
  plugin      Manage plugins
  scan*       Docker Scan (Docker Inc., v0.21.0)
  secret      Manage Docker secrets
  service     Manage services
  stack       Manage Docker stacks
  swarm       Manage Swarm
  system      Manage Docker
  trust       Manage trust on Docker images
  volume      Manage volumes

Commands:
  attach      Attach local standard input, output, and error streams to a running container
  build       Build an image from a Dockerfile
  commit      Create a new image from a container's changes
  cp          Copy files/folders between a container and the local filesystem
  create      Create a new container
  diff        Inspect changes to files or directories on a container's filesystem
  events      Get real time events from the server
  exec        Run a command in a running container
  export      Export a container's filesystem as a tar archive
  history     Show the history of an image
  images      List images
  import      Import the contents from a tarball to create a filesystem image
  info        Display system-wide information
  inspect     Return low-level information on Docker objects
  kill        Kill one or more running containers
  load        Load an image from a tar archive or STDIN
  login       Log in to a Docker registry
  logout      Log out from a Docker registry
  logs        Fetch the logs of a container
  pause       Pause all processes within one or more containers
  port        List port mappings or a specific mapping for the container
  ps          List containers
  pull        Pull an image or a repository from a registry
  push        Push an image or a repository to a registry
  rename      Rename a container
  restart     Restart one or more containers
  rm          Remove one or more containers
  rmi         Remove one or more images
  run         Run a command in a new container
  save        Save one or more images to a tar archive (streamed to STDOUT by default)
  search      Search the Docker Hub for images
  start       Start one or more stopped containers
  stats       Display a live stream of container(s) resource usage statistics
  stop        Stop one or more running containers
  tag         Create a tag TARGET_IMAGE that refers to SOURCE_IMAGE
  top         Display the running processes of a container
  unpause     Unpause all processes within one or more containers
  update      Update configuration of one or more containers
  version     Show the Docker version information
  wait        Block until one or more containers stop, then print their exit codes

Run 'docker COMMAND --help' for more information on a command.

To get more help with docker, check out our guides at https://docs.docker.com/go/guides/

make[1]: *** [Makefile:14: precommit_fix] Error 125
make[1]: Leaving directory '/home/lars/magma/docs/readmes'
make: *** [Makefile:10: precommit_fix] Error 2
```

Reply:
Test passed locally (and a similar command in the GitHub tests, https://github.com/magma/magma/actions/runs/3885179010/jobs/6628728757#step:4:14).
Did you upgrade your machine to V2 yet?

Reply:
No, that is why I wrote the comment below - ignore these comments here.

Reply:
After following [this install](https://docs.docker.com/desktop/install/ubuntu/) it works.